Applications/web_portal/web_app/library/source/pages/dashboard_page/notification_button.tsx

```diff
@@ -10,15 +11,29 @@ export interface Properties {
   isOpen: boolean;
 }
 
-export interface State {}
-
-/** Displays the notification button. */
-export class NotificationButton extends React.Component<Properties, State> {
-  constructor(properties: Properties) {
-    super(properties);
+/** Displays a notification button. */
+export class NotificationButton extends React.Component<Properties> {
+  public constructor(props: Properties) {
+    super(props);
   }
-
   public render(): JSX.Element {
-    return null;
+    const STYLE = StyleSheet.create({
+      button: {
+        width: '15px',
+        height: '20px',
+        border: 'none',
+        outline: 0,
+        padding: 0,
+        backgroundColor: 'rgba(0, 0, 0, 0)',
```

Comment:
What happens if you remove the background color?

Reply:
It will inherit from the parent so nothing in our case.

Reply:
Similarly then it should be safe to get rid of this background color right?

Reply:
Yup!

Reply:
sorry, correction. Buttons have a default background color I need to override.